generate/generate.go

```diff
@@ -956,11 +956,16 @@ func (g *Generator) SetupPrivileged(privileged bool) {
 		}
 		g.initSpecLinux()
 		g.initSpecProcessCapabilities()
-		g.spec.Process.Capabilities.Bounding = finalCapList
-		g.spec.Process.Capabilities.Effective = finalCapList
-		g.spec.Process.Capabilities.Inheritable = finalCapList
-		g.spec.Process.Capabilities.Permitted = finalCapList
-		g.spec.Process.Capabilities.Ambient = finalCapList
+		g.spec.Process.Capabilities.Bounding = []string{}
```

Comment:
Ah,  how about taking advantage of ClearProcessCapabilities()?

Reply:
fixed